tools/keycloak/sync/Syncronizer/SyncFactory.cs

```diff
@@ -425,12 +425,12 @@ private async Task<KModel.UserModel> GetUserAsync(PModel.UserModel user)
             try
             {
                 // Make a request to keycloak to find a matching user.
-                var response = await _keycloakManagementClient.HandleRequestAsync<ResponseWrapperModel<IEnumerable<KModel.UserModel>>>(HttpMethod.Get, $"{_keycloakManagementClient.GetEnvUri()}/idir/users?guid={user.GuidIdentifierValue.ToString().Replace("-", string.Empty)}");
-                if (response.Data.Count() > 1)
+                var users = await _keycloakService.GetUsersAsync(user.GuidIdentifierValue);
```

Comment:
this all looks good, using the repository should ensure consistency across the two projects.